<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShugoConsoleWidget</class>
 <widget class="QWidget" name="ShugoConsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShugoConsole</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="bannerLabel">
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
     <property name="locale">
      <locale language="French" country="France"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="ShugoConsole.qrc">:/ShugoConsole/banner.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="helpLabel">
       <property name="locale">
        <locale language="French" country="France"/>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Short notice&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Settings values (on the right side) are enforced in the game when the boxes (on the left side) are checked.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="defaultsLayout">
       <item>
        <widget class="QCheckBox" name="runOnStartupCheck">
         <property name="text">
          <string>Run on Startup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="defaultsSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="resetDefaultsButton">
         <property name="text">
          <string>Reset Default Values</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="consoleSettingsGroupBox">
       <property name="title">
        <string>Console Settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QFormLayout" name="consoleSettingsLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="consoleCheck">
            <property name="text">
             <string>CryEngine Console</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="consoleCombo">
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="consoleSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>71</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="cameraSettingsGroupBox">
       <property name="title">
        <string>Camera Settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QFormLayout" name="cameraSettingsLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="fovCheck">
            <property name="text">
             <string>Field of View (degrees)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="fovSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="minimum">
             <double>60.000000000000000</double>
            </property>
            <property name="maximum">
             <double>170.000000000000000</double>
            </property>
            <property name="value">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cammaxCheck">
            <property name="text">
             <string>Maximum Camera Distance (meters)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="cammaxSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="minimum">
             <double>5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>50.000000000000000</double>
            </property>
            <property name="value">
             <double>28.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="cameraSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="chatLogSettingsGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Chat Log Settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QFormLayout" name="chatLogSettingsLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chatLogCheck">
            <property name="text">
             <string>Chat Log</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="chatLogCombo">
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="chatLogSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="advancedSettingsGroupBox">
       <property name="title">
        <string>Advanced Graphics Settings</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QFormLayout" name="advancedSettingsLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QCheckBox" name="tripleBufferingCheck">
            <property name="text">
             <string>Triple Buffering</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="tripleBufferingCombo">
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="anisotropicCheck">
            <property name="text">
             <string>Anisotropic Filtering</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="anisotropicCombo">
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="maxfpsCheck">
            <property name="text">
             <string>FPS Limit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="maxfpsCombo">
            <item>
             <property name="text">
              <string>Enabled</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Disabled</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="advancedSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>67</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="processesGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Managed Aion Processes</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListView" name="processesView"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ShugoConsole.qrc"/>
 </resources>
 <connections/>
</ui>
